packages/odyssey-react-mui/src/Autocomplete.tsx

```diff
@@ -555,6 +561,21 @@ const Autocomplete = <
     [onInputChangeProp],
   );
 
+  const filterOptions: MuiAutocompleteProps<
+    OptionType,
+    HasMultipleChoices,
+    undefined,
+    IsCustomValueAllowed
+  >["filterOptions"] = useCallback(
+    (options: OptionType[], { inputValue }: { inputValue: string }) => {
+      if (getOptions) {
+        return getOptions(options, inputValue);
+      }
+      return options;
```

Comment:
@mohamed-okta This will just return all the options every time. Will override/break MUI's default filtering logic

Reply:
Ah yes I thought so at first but then ignored it when it was working, but that's because I was using server-side search as well, will fix